src/en/marinmoe/src/eu/kanade/tachiyomi/animeextension/en/marinmoe/MarinMoe.kt

```diff
@@ -155,7 +157,7 @@ class MarinMoe : ConfigurableAnimeSource, AnimeHttpSource() {
             val parsedDate = parseDate(it.release_date)
             if (parsedDate.time != -1L) episode.date_upload = parsedDate.time
 
-            episodes.add(episode)
+            if (!(!includeSpecial && it.type == 2)) episodes.add(episode)
```

Comment:
it just looks nicer this way:
```suggestion
            if (includeSpecial || it.type != 2) episodes.add(episode)
```
boolean algebra :)